subsys/modem/modem_stats.c

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright (c) 2024 Trackunit Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifdef _POSIX_VERSION
+#undef _POSIX_VERSION
+#endif
+#define _POSIX_VERSION 200809L
```

Comment:
Curious, what macro/symbol is it actually for?

Reply:
`strncpy` is a POSIX API, without declaring this, <string.h> doesn't expose it